prov/efa/src/rxr/rxr_op_entry.h

```diff
@@ -162,94 +175,24 @@ struct rxr_rx_entry {
 	 */
 	struct dlist_entry multi_recv_consumers;
 	struct dlist_entry multi_recv_entry;
-	struct rxr_rx_entry *master_entry;
-
+	struct rxr_op_entry *master_entry;
+	struct fi_msg *posted_recv;
 	struct rxr_pkt_entry *unexp_pkt;
 	char *atomrsp_data;
+	/* end of RX related variables */
 
-	/* linked with rx_entry_list in rdm_peer */
-	struct dlist_entry peer_entry;
-
-	/* linked with rx_entry_list in rxr_ep */
-	struct dlist_entry ep_entry;
-#if ENABLE_DEBUG
-	/* linked with rx_pending_list in rxr_ep */
-	struct dlist_entry rx_pending_entry;
-#endif
-};
-
-struct rxr_tx_entry {
-	/* Must remain at the top */
-	enum rxr_x_entry_type type;
-
-	uint32_t op;
-	fi_addr_t addr;
-	struct rdm_peer *peer;
-
-	/*
-	 * freestack ids used to lookup tx_entry during ctrl pkt recv
-	 */
-	uint32_t tx_id;
-	uint32_t rx_id;
-
-	uint32_t msg_id;
-
-	uint64_t tag;
-
+	/* the following variables are for TX operation only */
+	uint64_t iov_mr_start;
+	uint64_t iov_index;
+	uint64_t iov_offset;
 	uint64_t bytes_acked;
 	uint64_t bytes_sent;
-	int64_t window;
-
-	uint64_t total_len;
-
-	enum rxr_tx_comm_type state;
-	struct rxr_queued_ctrl_info queued_ctrl;
-
-	uint64_t fi_flags;
-	uint64_t rxr_flags;
-
-	size_t iov_count;
-	size_t iov_index;
-	size_t iov_offset;
-	struct iovec iov[RXR_IOV_LIMIT];
-
-	uint64_t rma_loc_rx_id;
-	uint64_t rma_window;
-	size_t rma_iov_count;
-	struct fi_rma_iov rma_iov[RXR_IOV_LIMIT];
-
-	/* App-provided reg descriptor */
-	void *desc[RXR_IOV_LIMIT];
-
-	/* atomic related variables */
-	struct rxr_atomic_hdr atomic_hdr;
-	struct rxr_atomic_ex atomic_ex;
-
-	/* Only used with mr threshold switch from memcpy */
-	size_t iov_mr_start;
-	struct fid_mr *mr[RXR_IOV_LIMIT];
-
-	struct fi_cq_tagged_entry cq_entry;
-
-	/* entry is linked with tx_pending_list in rxr_ep */
-	struct dlist_entry entry;
-
-	/* queued_ctrl_entry is linked with tx_queued_ctrl_list in rxr_ep */
-	struct dlist_entry queued_ctrl_entry;
-
-	/* queued_rnr_entry is linked with tx_queued_rnr_list in rxr_ep */
-	struct dlist_entry queued_rnr_entry;
-
-	/* Queued packets due to TX queue full or RNR backoff */
-	struct dlist_entry queued_pkts;
-
-	/* peer_entry is linked with tx_entry_list in rdm_peer */
-	struct dlist_entry peer_entry;
-
-	/* linked with tx_entry_list in rxr_ep */
-	struct dlist_entry ep_entry;
+	/* end of TX only variables */
 };
 
+#define rxr_tx_entry rxr_op_entry
```

Comment:
Since we already merge tx/rx into `op`, why do we still want to define these two macros?